addons/web_editor/static/src/js/wysiwyg/widgets/color_palette.js

```diff
@@ -71,7 +73,7 @@ const ColorPaletteWidget = Widget.extend({
      * @override
      */
     start: function () {
-        const res = this._super.apply(this, arguments);
+        const proms = [this._super.apply(this, arguments)];
```

Comment:
Feels like you can just await without any slowdown here. Instead of building a proms array then waiting for all